src/utils/canSkipApproval.ts

```diff
@@ -3,14 +3,23 @@ import { isSyncDomain } from '@src/background/services/network/utils/getSyncDoma
 import { isActiveTab } from './isActiveTab';
 import { runtime } from 'webextension-polyfill';
 
+type SkipApprovalOptions = {
+  allowInactiveTabs?: boolean;
```

Comment:
I went with dev having to explicitly say `allowInactiveTabs` rather than making the `tabId` optional. It feels less error-prone this way.